meta-lxatac-bsp/recipes-bsp/barebox/files/patches/0102-ARM-stm32mp-add-Linux-Automation-TAC-board.patch

```diff
@@ -346,43 +346,6 @@ index 000000000000..5a07ad7996e6
 +	};
 +};
 +
-+/*
-+ * There are two bindings for pin bias:
-+
-+ * - The pinctrl binding, which differs from controller to controller,
-+ *   but is capable of representing controller specific attributes like
-+ *   pull ups of varying strength. This is used via
-+ *   pinctrl-names/pinctrl-[0-9]+ DT properties
-+ *
-+ * - Or-ing GPIO_PULL_UP or GPIO_PULL_DOWN into the GPIO flags after the
-+ *   phandle, e.g.  gpios = <&gpioe 7 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
-+ *   Thix can expose a generic interface, but doesn't allow fine tuning.
-+ *
-+ * Not all Linux drivers support the second binding and barebox doesn't
-+ * support it all; It requires pinctrl drivers to expose to the core
-+ * a way to configure pull ups/pull downs, but we don't export this info.
-+ * Instead drivers are called with the device tree node in action and
-+ * instructed to apply it.
-+ *
-+ * We can add support for this binding for STM32 only as first step, but
```

Comment:
Why is this being removed? v2024.08.0 doesn't support the new binding either. I [added support recently](https://github.com/barebox/barebox/commit/269dad50c259d2685249d533bc17946cca5001bf), but it is slated for inclusion in v2024.10.0.

Reply:
It looks like the respective `fixup!` commits did not make it to the `v2024.03.0/customers/lxa/tac` branch or got force pushed away (sounds like something I would do by accident tbh).
I've made sure to push them to `v2024.03.0/customers/lxa/tac` again and have also included them in the last update of this PR.